lte/cloud/go/services/lte/analytics/calculations/state.go

```diff
@@ -223,11 +225,7 @@ func (x *SiteMetricsCalculation) Calculate(prometheusClient query_api.Prometheus
 			for _, pkg := range status.PlatformInfo.Packages {
 				if pkg.Name == "magma" {
 					if pkg.Version != "" {
```

Comment:
Since there's no else statement, just condense this logic into one line:
```
if pkg.Name == "magma" && pkg.Version != "" {
  magmaGatewayVersion[ent.PhysicalID] = pkg.Version
}
```